docs/api/core/shareLatest.md

```diff
@@ -5,23 +5,23 @@ title: shareLatest()
 An RxJS [pipeable operator] which multicasts the source stream and replays the
 latest emitted value.
 
+It's a utility function kept for historical purposes. Since RxJS@^7.0.0 released, it's equivalent to:
+
 ```ts
-function shareLatest<T>(): MonoTypeOperatorFunction<T>
+import { share } from 'rxjs/operators';
+
+function shareLatest<T>() {
+  return share<T>({
+    connector: () => new ReplaySubject(1)
```

Comment:
I thought about this - `resetOnError`, `resetOnComplete` and `resetOnRefCountZero` are all `true` by default on [RxJS](https://github.com/ReactiveX/rxjs/blob/master/src/internal/operators/share.ts#L145).

I still wondered whether to put it to make it explicit, or whether to leave it out, since it's what's _actually_ equivalent to (it's a `share` but with a ReplaySubject). Maybe making it explicit it's a bit more complete?

Reply:
Nah! You are right, there is no need to be explicit here. I got mixed up b/c the defaults used for `shareReplay` are different, but we are not mentioning `shareReplay` anywhere here, so yep, please disregard my comment.